src/Dto.fs

```diff
@@ -21,12 +21,19 @@ let deserialiseToAbsencesDto json =
 type SquadDto = 
     { Squad_5Gqot : string option }
 type EmployeeWorkDetailsDto = 
-    { Department : string
-      Custom: SquadDto }
+    {
```

Comment:
Fsharp convention is to have braces open on the same line as the code

Reply:
ooh, I didn't know that!
thanks Fred